roles/ceph-common/tasks/main.yml

```diff
@@ -11,6 +11,23 @@
        cache_valid_time=3600
   with_items: ceph.debian_pkgs
 
+- name: create ceph log directory
+  file: dest=/var/log/ceph
+        state=directory
+        owner=root
+        group=adm
+        mode=2750
+
+- name: Change ceph log directory permissions with acl
```

Comment:
So the `acl` task was how we were enforcing the inherited file permissions, but that doesn't appear to be working for ceph... Any ideas as to why? It works everywhere else.